packages/block-library/src/navigation/index.js

```diff
@@ -47,23 +47,23 @@ export const settings = {
 				name: 'core/navigation-link',
 				attributes: {
 					// translators: 'Home' as in a website's home page.
-					label: __( 'Home' ),
+					label: __('Home'),
 					url: 'https://make.wordpress.org/',
 				},
 			},
 			{
 				name: 'core/navigation-link',
 				attributes: {
 					// translators: 'About' as in a website's about page.
-					label: __( 'About' ),
+					label: __('About'),
 					url: 'https://make.wordpress.org/',
 				},
 			},
 			{
 				name: 'core/navigation-link',
 				attributes: {
 					// translators: 'Contact' as in a website's contact page.
-					label: __( 'Contact' ),
+					label: __('Contact'),
```

Comment:
Spaces should not be removed as per the WordPress Coding Standards
```suggestion
					label: __( 'Contact' ),
```